ga4gh/frontend.py

```diff
@@ -9,15 +9,19 @@
 
 import os
 import traceback
+import datetime
 
 import flask
 import flask.ext.api as api
```

Comment:
Each of these tools can be configured to include or exclude certain files, so I don't think that's a reason not to do it.